app/src/main/kotlin/com/google/samples/apps/nowinandroid/ui/NiaApp.kt

```diff
@@ -71,15 +68,18 @@ import com.google.samples.apps.nowinandroid.core.designsystem.component.NiaTopAp
 import com.google.samples.apps.nowinandroid.core.designsystem.icon.NiaIcons
 import com.google.samples.apps.nowinandroid.core.designsystem.theme.GradientColors
 import com.google.samples.apps.nowinandroid.core.designsystem.theme.LocalGradientColors
-import com.google.samples.apps.nowinandroid.feature.settings.SettingsDialog
-import com.google.samples.apps.nowinandroid.navigation.NiaNavHost
+import com.google.samples.apps.nowinandroid.core.navigation.NiaNavKey
+import com.google.samples.apps.nowinandroid.feature.bookmarks.impl.navigation.LocalSnackbarHostState
+import com.google.samples.apps.nowinandroid.feature.search.api.navigation.navigateToSearch
+import com.google.samples.apps.nowinandroid.feature.settings.api.SettingsDialog
+import com.google.samples.apps.nowinandroid.navigation.NiaNavDisplay
 import com.google.samples.apps.nowinandroid.navigation.TopLevelDestination
-import kotlin.reflect.KClass
-import com.google.samples.apps.nowinandroid.feature.settings.R as settingsR
+import com.google.samples.apps.nowinandroid.feature.settings.api.R as settingsR
 
 @Composable
 fun NiaApp(
     appState: NiaAppState,
+    entryProviderBuilders: Set<@JvmSuppressWildcards EntryProviderBuilder<NiaNavKey>.() -> Unit>,
```

Comment:
nit: Is `@JvmSuppressWildcards` required even though this isn't the injection site?

Reply:
Removed all @JvmSuppressWildcards from declarations without injection.